partiql-parser/src/lalr/partiql.lalrpop

```diff
@@ -466,16 +476,20 @@ ExprPrecedence04: ast::Expr = {
 }
 
 ExprPrecedence03: ast::Expr = {
-    "+" <r:ExprPrecedence02> =>
+    "+" <r:ExprPrecedence03> =>
        ast::Expr{ kind: ast::ExprKind::Pos( ast::Pos{ expr: Box::new(r) } ) },
-    "-" <r:ExprPrecedence02> =>
+    "-" <r:ExprPrecedence03> =>
```

Comment:
Just wondering why these precedences were changed?

Reply:
I noticed that this rule was not self-recursive, so I fixed it.

This allows unary minus & plus to nest like the unary "NOT". 

e.g. before `--5` would have resulted in a parse error, but now it will succeed.

Similar to the "NOT" rule: 
```rust
ExprPrecedence11: ast::Expr = {
    "NOT" <r:ExprPrecedence11> =>
       ast::Expr{ kind: ast::ExprKind::Not( ast::Not{ expr: Box::new(r) } ) },
    <ExprPrecedence10>,
}
```